logstash-core/src/main/java/JrubyEventExtService.java

```diff
@@ -1,11 +0,0 @@
-import org.jruby.Ruby;
-import org.jruby.runtime.load.BasicLibraryService;
-import org.logstash.ext.JrubyEventExtLibrary;
-
-public final class JrubyEventExtService implements BasicLibraryService {
```

Comment:
for my own education what did this do ?

Reply:
@jakelandis this was automatically (it scans all classes to find these `LibraryService` instances) invoked by JRuby at some point during its startup (a point later than the classload of `RubyUtil`) to give us the ability to then `require "jruby_event_ext"` (obviously not fun in our case because of the cyclic dependency and us only being able to run this after the `Logstash::Json` module was loaded, more fun if we control the load order manually now :)).